doc/docker_quick_start.rst

```diff
@@ -0,0 +1,63 @@
+Docker Quick Start
+==========================
+
+How to add a new asset to Ralph and discover information about the added device in Docker Container
```

Comment:
For example. 

Let's learn Ralph by example. Throughout this tutorial, we’ll walk you through:

- adding asset information, licenses and support
- scanning asset details and configuration
- access billing information